src/Websockettest.js

```diff
@@ -73,12 +78,19 @@ const RMBTTest = (function() {
     /**
      *
      * @param {RMBTTestConfig} rmbtTestConfig
+     * @param {RMBTControlServerCommunication} rmbtControlServer
+     * @param {Logger} newLogger
      * @returns {}
      */
-    function RMBTTest(rmbtTestConfig) {
+    function RMBTTest(rmbtTestConfig, rmbtControlServer, newLogger) {
```

Comment:
That is the preparation to use different logger implementations, depending on your needs. (Like one that has a log level or uses a Dom-element to show messages on the website). Later, other classes should use this as a factory to create their own instances - like `Logger.getLogger()` in java